docs/myriad-configuration.md

```diff
@@ -23,6 +23,9 @@ frameworkFailoverTimeout: 43200000
 # Myriad's mesos framework name.
 frameworkName: MyriadAlpha
 
+# Myriad's mesos framework role.
+frameworkRole: someRoleName
```

Comment:
@everpeace I think we should add "frameworkRole" property to https://github.com/mesos/myriad/blob/phase1/src/main/resources/myriad-config-default.yml as well. The value can be left empty.